components/ionFooterBar/ionFooterBar.js

```diff
@@ -1,11 +1,87 @@
-Template.ionFooterBar.rendered = function () {
-  Session.set('hasFooter', true);
-};
+//Template.ionFooterBar.rendered = function () {
```

Comment:
Tell me when that is done. After that, I suggest squashing all the commits. 

So all the feature changes are in one commit. Before you do that, so you don't ruin your `master` branch, I suggest doing 

git checkout -b 'wip-ionFooter-Upgrade'

And play squash that branch. If you don't know how to do that, tell me, and I'll do it with you later.